### ISSUE TITLE
Add a guide about importing module

### ISSUE THREAD

**Issue by lumtis** [ignite/modules#26]
It doesn't seem there is a comprehensive guide somewhere about the changes in `app.go` you need to make to import a new module.

Add such a guide under this repository, like `docs/importing.md`. Add redirection to this guide in `readme.md`

**Comment by amedumer** [comment]
Hey @lumtis  , can I work on this task?

**Comment by lumtis** [comment]
> Hey @lumtis , can I work on this task?

@Pantani

**Comment by Pantani** [comment]
@amedumer, yes, sure!! You can take it. Any questions or concern regarding the task?
